relay-server/src/actors/store.rs

```diff
@@ -563,16 +563,50 @@ impl StoreService {
         let payload = item.payload();
 
         for bucket in Bucket::parse_all(&payload).unwrap_or_default() {
+            let (flat_type, flat_value) = match bucket.value {
+                BucketValue::Distribution(value) => (
+                    ingest_metrics_v1::IngestMetricType::D,
```

Comment:
i believe it would require us to write custom rust types for the bucketvalue and then patch them into the generated code using https://docs.rs/typify/latest/typify/struct.TypeSpaceSettings.html#method.with_replacement -- i don't think that's worth it